### PR TITLE
Build: Release PR v2.6.18 Release

### PR DESCRIPTION
**Pull request by vrundakansara**
### Description
- Final release PR. Please check all the checklist below before merging.
---
### Checklist:
- [x] Finalize readme.txt
- [x] Add doc links in the readme.txt
- [x] Check if Gutenberg Blocks/Astra Notices library is updated to latest - Run `composer update`
- [x] Run to update packages        : `npm audit fix`
- [x] Run to produce rtl files : `grunt rtl`
- [x] Run to update the Template Library    : `wp astra-sites sync --force`
- [x] PHPCS - vendor/bin/phpcs
- [x] Run to update version number  : `grunt version-bump --ver=<version-number>`
- [x] Update - Tested Upto - readme.txt
- [x] Update readme.txt file and then run: `grunt readme`
- [x] Generate POT file             : `grunt i18n`
- [ ] Add draft Release notes on github
- [ ] Add draft changelogs to wpastra.com
- [ ] Open PR to backport branch to `next-release`
---

- Improvement: Avoided loading a few scripts in the customizer screen.
- Improvement: Free Images from Pixabay can be downloaded from the main grid too with the use of a download icon.
- Fix: Updated the Quick Links to Upgrade the plugin, page builder wise.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
           tools: composer
 
       - name: Install dependencies
-        run: composer install --prefer-dist --no-suggest --no-progress
+        run: composer config github-oauth.github.com ${{ secrets.PRIVATE_ACCESS_TOKEN }} && composer install --prefer-dist --no-suggest --no-progress
```
Comment (maheshwaghmare) [inline .github/workflows/ci.yml:21]: Is this not required for the premium plugins [ci.yml](https://github.com/brainstormforce/astra-pro-sites/blob/master/.github/workflows/ci.yml)?

**Review by maheshwaghmare — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (vrundakansara) [reply .github/workflows/ci.yml:21]: It is required @maheshwaghmare - We do need `phpcs` here too.

**Review by amithire1201 — APPROVED** [review]

File: inc/lib/gutenberg-templates/classes/class-ast-block-templates.php [hunk 2]
```diff
@@ -390,7 +390,7 @@ public function template_assets() {
 						'wpforms_status'          => $this->get_plugin_status( 'wpforms-lite/wpforms.php' ),
 						'gutenberg_status'        => $this->get_plugin_status( 'gutenberg/gutenberg.php' ),
 						'_ajax_nonce'             => wp_create_nonce( 'ast-block-templates-ajax-nonce' ),
-						'button_text'             => esc_html__( 'Starter Templates', 'ast-block-templates' ),
+						'button_text'             => esc_html__( 'Starter Templates', 'ast-block-templates', 'astra-sites' ),
 						'display_button_logo'     => true,
```
Comment (maheshwaghmare) [inline inc/lib/gutenberg-templates/classes/class-ast-block-templates.php:394]: Remove extra textdomain.

File: readme.txt [hunk 3]
```diff
@@ -149,6 +149,11 @@ We are open to suggestions and would love to work on topics that our users are l
 
 == Changelog ==
 
+v2.6.18 - 27-July-2021
```
Comment (maheshwaghmare) [inline readme.txt:152]: Change date

**Pull request merged**